deeppavlov/configs/squad/squad_torch_bert.json

```diff
@@ -158,16 +158,18 @@
   },
   "metadata": {
     "variables": {
+      "LOWERCASE": true,
+      "TRANSFORMER": "bert-base-uncased",
       "ROOT_PATH": "~/.deeppavlov",
       "DOWNLOADS_PATH": "{ROOT_PATH}/downloads",
-      "MODELS_PATH": "{ROOT_PATH}/models/squad_torch_bert"
+      "MODELS_PATH": "{ROOT_PATH}/models",
+      "MODEL_PATH": "{MODELS_PATH}/squad_torch_bert/{TRANSFORMER}"
     },
     "download": [
       {
-        "url": "http://files.deeppavlov.ai/deeppavlov_data/squad_torch_bert_v0.tar.gz",
+        "url": "https://files.deeppavlov.ai/v1/squad/squad_torch_bert.tar.gz",
```

Comment:
Is this file with new pretrained bert using transformers 4.6.0? I've also trained squad torch bert for compatibility, here is url: http://files.deeppavlov.ai/deeppavlov_data/squad_torch_bert_v1.tar.gz
I'll change url in my config to yours and remove my archive, ok?

Reply:
Evaluation scores should be also considered to make this decision.

Reply:
Yes, it's a model that was retrained 4.6.0, because the weights trained on the old version couldn't be loaded properly. And it performs a bit worse, although it's not clear why. I'll have to check whether the weights that you're suggesting are going to load/work on my branch first.